### ISSUE TITLE
what about the training progress

### ISSUE THREAD

**Issue by peyer** [tj27-vkr/RCNN-Vehicle-Tracking-Lane-Detection#1]
@tj27-vkr Hi, I am amazed of you reimplementation of Mask-RCNN with the land detection branch and road segmentation branch added. So, how have you trained your mask rcnn project? especially  what's your training dataset?

**Comment by tj27-vkr** [comment]
I have used a pre-trained model which was trained on MS COCO object detection dataset which can be found [here](http://cocodataset.org/#home). There will be multiple classes in the dataset which was reduced to detect only vehicles.

**Comment by peyer** [comment]
@tj27-vkr Sorry, but at the first time, I just thought that you have added a lane detection branch to mask rcnn just as mask generate branch and 2d bbox generate bbox branch , it seems that you  used  sobel  instead of cnn to detect lane.

**Comment by tj27-vkr** [comment]
Yes. Using sobel filter was enough to detect the lanes with legible markings and to use a CNN to detect that also will increase the overhead. Right now, the model takes about 15s for detection in a single frame on NVIDIA 820 GPU.
